diff --git a/features.module b/features.module
index 172737e..5a42eb5 100644
--- a/features.module
+++ b/features.module
@@ -661,6 +661,10 @@ function features_get_info($type = 'module', $name = NULL, $reset = FALSE) {
     $files = system_rebuild_module_data();
 
     foreach ($files as $row) {
+      // Remove modification timestamp, added in Drupal 7.33.
+      if (isset($row->info['mtime'])) {
+        unset($row->info['mtime']);
+      }
       // Avoid false-reported feature overrides for php = 5.2.4 line in .info file.
       if (isset($row->info['php'])) {
         unset($row->info['php']);
